src/styles/InstructorLandingStyles/index.less

```diff
@@ -14,6 +14,14 @@
 .il__heading {
   font-family: 'Staatliches', 'cursive';
 }
+.il__heading_inner{
+  display: flex;
+  justify-content: center;
+}
+.il__browseBtn__container {
+  display: flex;
+  flex-direction: row;
```

Comment:
Whenever you assign flex to the direction it defaults to row, so the "flex-direction: row" is just a little extra code.